### DISCUSSION TITLE
[Help Wanted] Are wav outputs of the python default models subject to Copyright?

### DISCUSSION THREAD

**Discussion by CryptoCashCashoo** [coqui-ai/TTS#1502] [General Q&A] [answered]
**🚀 Feature Description**
I've installed the python lib, and I see a lot of diferent models.
Are the produced outputs with the default models subjects to copyright ?
I've read the license carefully but couldn't find information about the produced archives.
Thanks.

**Comment by erogol** [comment, marked as answer]
Currently, you can see model licenses in .model.json file. Outputs are subject to the models' licenses.

**Reply by CryptoCashCashoo** [reply]
Thanks for the answer, the majority of those models are TBD (to be discussed).
Can I use them for Commercial use @erogol ?

**Reply by erogol** [reply]
Check again there is not TBD anymore

**Comment by suflaj** [comment]
Regarding this, the `.model.json` file does list licenses, but the actual LICENSE files, which according to these licenses must be redistributed alongside the licensed material, are nowhere to be found, not even in the model files.

This would mean that the current way of distributing these models is in breach of these licenses, or they are simply void. If I wanted to distribute the .wav files as a part of dataset generated by these models, what should I do?

**Reply by erogol** [reply]
yes, probably we should download the license files along with the model files. But you can do whatever you want with the outputs. If I am not missing anything, there is no model preventing such action in the zoo.

**Reply by suflaj** [reply]
> yes, probably we should download the license files along with the model files. But you can do whatever you want with the outputs. If I am not missing anything, there is no model preventing such action in the zoo.

This, according to the LICENSEs, is not true. The LICENSEs enable you to do anything, but the require you to also keep the notices and the original LICENSE.

Apache 2.0 clause 4.a states:
> You must give any other recipients of the Work or Derivative Works a copy of this License

BSD-3 clause 2 states:
> Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.

 Therefore, distributing work without the original LICENSE, even if it is derivative, would constitute a breach of both Apache 2.0 and BSD-3.

**Reply by erogol** [reply]
I see. Thanks for the flag. We'll fix it.

**Reply by dennykurniawan48** [reply]
Hello, do we have an update on this issue? Thanks in advance.
